src/components/sponsors/styles.module.scss

```diff
@@ -0,0 +1,170 @@
+.sponsors {
+    .background{
+        background-image: linear-gradient(to bottom, #196799, #27A4B6);
+        position: relative;
+        min-height: 200vh;
+        z-index: 2;
+        margin-bottom: 0;
+        @media only screen and (max-width: 768px) {
+             min-height: 100vh;
+          }
+          @media only screen and (min-width: 1000px) {
+            min-height: 250vh;
+         }
+         
+          
+
+        .rect{
+            position: absolute;
+            width: 57%;
+            height: auto;
+            left: 25%;
+            top: 14%; //21%
+            text-align: center;
+            border-radius: 40px;
+            transform: rotate(2.49deg);
+    
+    
+    background: #1D2644;
+    border-radius: 20px;
+    transform: matrix(1, -0.05, 0.02, 1, 0, 0);
+    
+    .h1{
+      
+        align-items: center;
+        justify-content: center;
+        font-family: 'Montserrat';
+        font-style: normal;
+        text-align: center;
+        margin: 26px;
+        
+        font-weight: 600;
+        font-size: 70px;
+        line-height: 85px;
+        transform: matrix(1, -0.05, 0.02, 1, 0, 0);
+        color: #C9D2E5;
+    }
+    }
+    
+    .emailBox {
+        position: absolute;
+        background-color:#0D387C8F;
+        width: auto;
```

Comment:
Instead of making the width `auto`, how about setting it to `90vw` so that it isn't the full width of the screen?